media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -229,10 +239,15 @@ pkg_postinst() {
 		elog "by commenting out the relevant command near the end of the file:"
 		elog "#\"/usr/bin/pipewire\" = { args = \"-c pipewire-pulse.conf\" }"
 		elog
-		elog "It is still necessary to manually enable PipeWire startup. Setup specific"
-		elog "instructions can be found at: https://wiki.gentoo.org/wiki/PipeWire"
+		ewarn "${EROOT}/etc/xdg/autostart/pipewire.desktop has been installed. Users of XDG"
+		ewarn "compliant desktops on OpenRC must not manually start pipewire anymore!"
+		ewarn "Users wishing to use PulseAudio must remove the file manually and add the"
```

Comment:
We do not advertise `INSTALL_MASK` feature. So please remove the last two `ewarn` messages.

Reply:
Right, I missed that during backporting. Sorry.